ops/deploy_remote.sh

```diff
@@ -2,11 +2,11 @@
 set -e
 
 IMAGE=${1:-nearprotocol/nearcore:0.1.1}
-PREFIX=${2:-testnet}
+PREFIX=${2:-testnet-${USER}-5}
```

Comment:
this seems unintended

Reply:
It is actually intended. We had some confusion locally in the team with who running which testnet. This should disambiguate.

Reply:
the 5 part is intended?

Reply:
That part was removed.